File: sort/Minimum.py
data = [0, 2, 1, 3, 4, 6, 8, 4, 9]

def get_minimum(data):
    min = 10000
    for i in range(len(data)):
        if(data[i] < min):
            min = data[i]
    print(f"The minimum is {min}")

get_minimum(data)

File: search/Minimum.java
package sort;

public class Minimum {
    public static void main(String[] args) {
        int[] data = {32, 12, 32, 43, 2, 1, 7, 43, 2, 5};

        int min = 10000;
        for(int i: data)
            if(i < min)
                min = i;
        System.out.println("The minimum number is " + min);
    }
}
